demisto_sdk/commands/validate/sdk_validation_config.toml

```
@@ -207,6 +207,7 @@ select = [
   "RM113",
   "RM114",
   "RM115",
+  "RM118",
```

```suggestion
  "RM103",
```
Please use RM103 and put it in its place.